CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 #### Changes
+* Node: Added LPUSHX and RPUSHX command([#1959](https://github.com/valkey-io/valkey-glide/pull/1959))
```

Comment:
Is there an order we need to follow here? Add to end of the list vs beginning?

Reply:
I don't think the order matters, the other changelog entries aren't necessarily in order